WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-ColHeader {
+	border-bottom:0px;
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskListViewItem@
+}
+
+.ZmTaskMultiView .DwtListView .RowDouble td {
+	@TaskListViewItemCheckbox@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskSubject {
+	@TaskListViewItemSubject@
 }
 
-.newTaskBannerSep {
-	border-bottom:2px solid #AAA;
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskDate {
+	@TaskListViewItemDate@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow {
+	@TaskListViewItemBottomContainer@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconOuterWrapper {
+	@TaskListViewItemBottomIconOuterWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconInnerWrapper {
+	@TaskListViewItemBottomIconInnerWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-Rows  .ImgCheckboxUnchecked-svg {
+	padding-top:32px;
```

Comment:
In the previous PR, I have used a common property for both checked icon and unchecked icon.
.ZmTaskMultiView .DwtListView .DwtListView-Rows  .RowDouble .ImgCheckboxUnchecked-svg,
.ZmTaskMultiView .DwtListView .DwtListView-Rows  .RowDouble .ImgCheckboxChecked-svg { 

And in the updated PR, I have mapped them to a skin property name, TaskReadingPaneRight-ListViewCheckBoxIcon , as asked by you.
Hope this helps.